Description: allow undici-related failures on riscv64 for the time being
Forwarded: not-yet
Last-Update: 2022-10-01
Author: Jérémy Lal <kapouer@melix.org>
--- a/test/parallel/parallel.status
+++ b/test/parallel/parallel.status
@@ -140,3 +140,7 @@
 test-http-upgrade-advertise: PASS, FLAKY
 test-tls-client-mindhsize: PASS, FLAKY
 test-tls-write-error: PASS, FLAKY
+
+[$arch==riscv64]
+test-fetch: PASS, FLAKY
+test-wasm-web-api: PASS, FLAKY
